src/manifolds/ProductManifold.jl

```diff
@@ -42,12 +42,22 @@ end
 
 const PRODUCT_BASIS_LIST_CACHED = [CachedBasis]
 
-"""
-    ProductMetric <: Metric
+@doc raw"""
+    ProductMetric{<:(Tuple{N,Metric} where N)} <: Metric
+
+A type to represent the product of `metrics` for a [`ProductManifold`](@ref).
```

Comment:
`metrics` here isn't a variable or type in this docstring, so it doesn't need the ticks
```suggestion
A type to represent the product of [`Metric`](@ref)s for a [`ProductManifold`](@ref).
```